cinema/tests/test_actor_api.py

```diff
@@ -16,9 +16,7 @@ def test_get_actors(self):
         response = self.client.get("/api/cinema/actors/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         actors_full_names = [actor["full_name"] for actor in response.data]
-        self.assertEqual(
-            sorted(actors_full_names), ["George Clooney", "Keanu Reeves"]
-        )
+        self.assertEqual(sorted(actors_full_names), ["George Clooney", "Keanu Reeves"])
```

Comment:
You may have run some linter which changed a lot of files, but it is hard to review it due to a lot of changes

Reply:
Yes I used black , next time will exlude tests files